Orange/widgets/visualize/utils/tree/skltreeadapter.py

```diff
@@ -236,7 +236,7 @@ def assign(node_id, indices):
                 feature_idx = self._tree.feature[node_id]
                 thresh = self._tree.threshold[node_id]
 
-                column = self.instances.X[indices, feature_idx]
+                column = self.instances.transform(self.domain).X[indices, feature_idx]
```

Comment:
Running transform in the `assign` function could be problematic since it's called recursively. If the tree is large, then this will be called many times. It would be better to cache the thing outside `assign`.

Also, could you add a test for this?